backend/mancala.py

```diff
@@ -28,15 +28,16 @@ def getHumanName(self):
         return input("Please input your name: ")
 
     def getNextMove(self):
-        try:
-            selection = int(input("%s Please input your next move (1 to 6): " % self.name))
-            if (selection < 1) or (selection > 6):
-                print("Input is out of range (1 to 6)")
-                sys.exit()
-            return selection-1
-        except ValueError:
-            print("Input is not an integer")
-            sys.exit()
+        while True:
```

Comment:
was there a reason for adding this into a while loop?

Reply:
I think the only thing this does in not kill the program when you input something wrong